<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProTra</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../stopwatch.png</normaloff>../stopwatch.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>750</width>
        <height>700</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="widget">
       <attribute name="icon">
        <iconset>
         <normaloff>../timer.png</normaloff>../timer.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Tracker</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>551</width>
          <height>280</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLineEdit" name="lineEditProject">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>EADS Sans</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="pushButtonStart">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>EADS Sans</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>START</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../play.png</normaloff>../play.png</iconset>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="pushButtonStop">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>EADS Sans</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>STOP</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../stop.png</normaloff>../stop.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QTableWidget" name="tableWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>514</width>
               <height>80</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>600</width>
               <height>200</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>EADS Sans</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <column>
              <property name="text">
               <string>Date</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Project</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Start</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>End</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Time on project</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButtonRemove">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>140</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>EADS Sans</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>REMOVE ROW</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../remove.png</normaloff>../remove.png</iconset>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButtonExport">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Palatino Linotype</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>EXPORT</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../export.png</normaloff>../export.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>279</y>
          <width>551</width>
          <height>281</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="plotlayout_2">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <property name="margin">
            <number>10</number>
           </property>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="widget">
       <attribute name="icon">
        <iconset>
         <normaloff>../plot-8.png</normaloff>../plot-8.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Bar chart</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>581</width>
          <height>511</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="plotlayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>100</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>2</x>
          <y>12</y>
          <width>466</width>
          <height>32</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="labelStart">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>From</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelEnd">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>To</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_2">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonFilter">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Filter</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>filter.png</normaloff>filter.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>618</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
